opennms-web-api/src/main/java/org/opennms/web/api/Util.java

```diff
@@ -164,9 +164,10 @@ protected static String substituteUrl(final HttpServletRequest request, final St
      */
     public static String getHostHeader(final HttpServletRequest request) {
         for (int i = 0; i < hostHeaders.length; ++i) {
-        	final String ret = request.getHeader(hostHeaders[i]);
-            if (ret != null) {
-                return ret;
+            // Get the first value in the header (support for proxy-chaining)
+            final Enumeration<String> headers = request.getHeaders(hostHeaders[i]);
+            if (headers.hasMoreElements()) {
+                return headers.nextElement();
```

Comment:
The difference is that ```getHeaders``` returns the values of the header already splitted whereas ```getHeader``` returns the concatinated format.